README.md

```diff
@@ -1,9 +1,48 @@
+[![Releases](https://img.shields.io/github/v/release/stuajnht/CSSS?include_prereleases)]()
+[![Maintenance](https://img.shields.io/maintenance/yes/2019.svg)]()
+[![Known Vulnerabilities](https://snyk.io/test/github/stuajnht/CSSS/badge.svg?targetFile=CSSS.sln)](https://snyk.io/test/github/stuajnht/CSSS?targetFile=CSSS.sln)
+[![Contributors](https://img.shields.io/github/contributors/stuajnht/CSSS.svg)]()
+[![Forks](https://img.shields.io/github/forks/stuajnht/CSSS.svg)]()
+[![Issues](https://img.shields.io/github/issues/stuajnht/CSSS.svg)](https://github.com/Stormy102/Lancer/issues)
+
 CyberSecurity Scoring System (CSSS) is a program to assist with training for the CyberPatriot / CyberCenturion competitions.
 
 While it is currently possible to create [your own practice images](https://www.uscyberpatriot.org/competition/training-materials/practice-images), there is only a scoring client for Windows computers. To assist with training competitors with Windows, Ubuntu and Debian Operating Systems, this program has been created to provide a scoring system for them.
 
+## Disclaimer
+This program is designed to score the state of a computer image against a pre-defined checklist of "issues". It is not designed to rate your computer security, provide security advice, or anything else that isn't related to the CyberPatriot / CyberCenturion competitions.
+
+## To-Do
+These are the tasks planned for each release version. Please note that these can change without warning, depending on how features are progressing.
+
+### Version 0.3
+* Perform various tasks when the '-p' argument is passed:
+  * ~~Set issue check files to be encrypted, so that images can be released but competitors can not find out what "issues" need to be fixed~~
+  * ~~Add files to the relevant Operating System startup folders, so that CSSS will start automatically:~~
+    * ~~[Windows shortcut](http://stackoverflow.com/a/19914018) in [Programdata startup folder](https://www.kiloroot.com/all-users-or-common-startup-folder-locations-launch-programs-at-window-login-windows-server-2008-r2-2012-2012-r2/)~~
+    * ~~[Linux](http://raspberrypi.stackexchange.com/a/5159)~~
+  * Email list of issues to be fixed when '--email' argument is passed
+  * Shutdown the computer to allow an image to be taken
+* Comment ~~and coding style~~ tidy-up
+
+### Version 0.4
+* Create scripts to install CSSS files into the correct locations (WinNT: root of `%SystemDrive%`, Linux: `/opt` directory)
+* Install additional software when CSSS is being installed to make sure all items are available for it to use
+* Create build scripts that install software that can be used when testing the checker system engine
+
+### Versions 0.5 onward
+* Create more check template files and relevant classes for the checker system engine to use (a new version for each check "category")
+* Create some example checker files with instructions to create images based around them with pre-made questions and scenarios
+

```

Comment:
Could the disclaimer and To-Do sections go back to their original locations please? It makes more sense for users to know how to run the program than what it currently can't do.